pom.xml

```diff
@@ -348,12 +356,36 @@
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.togglz</groupId>
+        <artifactId>togglz-spring-boot-common-autoconfigure</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.togglz</groupId>
+        <artifactId>togglz-spring-boot1-autoconfigure</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.togglz</groupId>
+        <artifactId>togglz-spring-boot2-autoconfigure</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+
       <dependency>
         <groupId>org.togglz</groupId>
         <artifactId>togglz-spring-boot-starter</artifactId>
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.togglz</groupId>
+        <artifactId>togglz-spring-actuator-boot-starter</artifactId>
```

Comment:
Not sure what this artifact is about, but there is a difference between `actuator` and `starter`. Actuator are basically exposing metrics and management functionality whereas starters are about adding features.

Reply:
Geez, you are right, I don't even know where that came from :D Definitely a mistake, there's no module with that name neither, I'll fix it.